hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/hip993/SystemFileExportsTest.java

```diff
@@ -463,6 +466,11 @@ private static VisibleItemsValidator nodeDetailsExportValidator(
         };
     }
 
+    private static byte[] getHexStringBytesFromBytes(final byte[] rawBytes) {
+        final String hexString = HexFormat.of().formatHex(rawBytes);
+        return Normalizer.normalize(hexString, Normalizer.Form.NFD).getBytes(UTF_8);
+    }
+
```

Comment:
Would it make sense to move this to a common module, e.g. hapi-utils?

Reply:
Possibly.  We're fixing an issue in release 0.56 here, but I fully support a follow-up PR to move this to hapi-utils and add even more thorough unit tests.